db/migrate/20190318190517_add_zone_to_service_templates.rb

```diff
@@ -0,0 +1,5 @@
+class AddZoneToServiceTemplates < ActiveRecord::Migration[5.0]
+  def change
+    add_column :service_templates, :zone, :bigint
```

Comment:
Shouldn't this be adding a reference? Not a column called `zone`?

Reply:
Oops, I changed it from ```add_reference``` to ```add_column``` and forgot renaming the field. 

Index seems not necessary so I selected ```add_column``` over ```add_reference```.  Seems there is no any place where we would query select  service templates by a zone.